[Ready For Review] Add chart webhook-receiver (#263)
rancher/system-charts: charts/rancher-monitoring/v0.1.3/README.md

> **Tip**: Only use for Rancher Monitoring!!!

## Changelog

* 0.1.3

>     * Added chart webhook-receiver to support some recipients those not directly supported by Prometheus. Such as DingTalk and Microsoft Teams, etc.

Reviewer: ```suggestion
    * Added chart webhook-receiver to support recipients that are not directly supported by Prometheus
```
